scripts/globals/weaponskills.lua

```diff
@@ -900,6 +902,10 @@ function getMultiAttacks(attacker, numHits)
             bonusHits = bonusHits + 2;
         elseif(chance < doubleRate + tripleRate + quadRate) then
             bonusHits = bonusHits + 1;
+        elseif(chance < oaThriceRate) then
```

Comment:
this isn't right, but after getting thorny to test for me, the whole loop isn't right either. for your case, it needs to be after assassins/warriors, and it should only be when i == 1 (the first hit)

i'll update the formula to be right, then you'll have to merge master into your branch again

Reply:
oh sorry, the position relative to assassins/warriors charge doesn't matter, thats just removing the effect but it still should be mainhand only